packages/example-app-base/src/components/App/App.tsx

```diff
@@ -64,16 +64,18 @@ export class App extends React.Component<IAppProps, any> {
       responsiveMode = ResponsiveMode.large;
     }
 
```

Comment:
Hey, this is equivalent! This function should be defined outside the `render` definition,

Reply:
Thanks @ThomasMichon , I moved the function outside of render and pushed the commit. Is that for performance?